### DISCUSSION TITLE
Implementing polls on dev.to

### DISCUSSION THREAD

**Discussion by wuz** [forem/forem#15896] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
There are many great types of content on dev.to, but I would love to see a way to gather information from the community. I think polls are a natural fit alongside things like #discuss topics and many other dev.to features.

**Describe the solution you'd like**
I'd love to see a tag and liquid tag for implementing polls. To begin with it might be worthwhile to test out the feature with a #poll tag and an embedded poll service. If that goes well, a built in poll feature could be very cool. That would prevent the need for a separate service and could lead to some interesting statistics around the types of questions and answers that are created.

**Describe alternatives you've considered**
As I said, using a third-party system is a good alternative to start with.

**Additional context**
N/A

**Comment by abraham** [comment]
I was just thinking about requesting/proposing polls last week. :+1:

**Comment by maestromac** [comment]
Thank you for this feature request! That would be a great feature to have and I bet @benhalpern already has some thoughts on this.

**Comment by heyphilllie** [comment]
> I was just thinking about requesting/proposing polls last week. :+1: 

Same over here! So yeah, this would be awesome. :heart_eyes:

**Comment by kay-is** [comment]
Anything happening here?

I just had the same idea after I saw that many people just Heart/Unicorn my posts without doing anything else with it.

Would be cool to have the option to add other "one-click-interactions" to this and maybe make it to be configurable as public/private so peoples opinions aren't affected by previous votes. Also, sometimes I just want a response and don't show it to everyone, especially if the outcome is negative.

**Comment by benhalpern** [comment]
https://github.com/thepracticaldev/dev.to/pull/3176 addresses the first big steps.

We are not all the way yet, as I left some of the functionality to be figured out and implemented.

Here is a post about this: https://dev.to/devteam/feedback-on-this-new-dev-feature-5cai

We should follow up with some additional sub-issues in terms of getting this feature finished-finished.

**Comment by gauthamp10** [comment]
We need polls @ dev.to

**Comment by cmgorton** [comment]
@benhalpern any more thoughts on this feature request. Is this something we are likely to implement in the near future?

**Comment by cmgorton** [comment]
Since this has been stale for awhile now I am going to go a head and close this issue. 
 We have a new process moving forward to address feature requests and especially those that have been in our repo for 1+ years without much traction.  
TLDR: we have new internal RFC process for feature requests. Any features that are more substantial and require more thought and detail will need to be discussed in [forem.dev](https://forem.dev/).

You can read more about our Internal RFC Process and forem.dev Discussions in [this post](https://forem.dev/foremteam/internal-rfc-process-and-forem-dev-discussions-3gl4).

**Comment by michael-tharrington** [comment]
I've just reopened this one as this is something that continues to be in demand.

**Comment by pandeybk** [comment]
Upvote for this ..
